### DISCUSSION TITLE
How to  Analyze ^GFA with Z64 format data?

### DISCUSSION THREAD

**Discussion by masternan** [BinaryKits/ZPLUtility#161] [General]
I use ZebraDesigner  to design label,and  then print it to prn file.In the prn file ,i find the string as following:
`^FO800,128^GFA,01536,01536,00016,:Z64:
eJztk71OwzAQx89EkYdKNRsLqhnZWDtUTYbyBjD3FRgZimypD4LEyEvUfQMG2DMiJg9Fiqo05uz4EpcFZoSlOvrpenf/+zDAb86iPMIT95EiU869JDxyz5t9wupzce3MwC3+RNUj3/kYh56F9vd8cA/Jl32AOtwZBWC2+9SRyTCNLKOjijw5SgPwQDLMsV/W2RmqW7K73MQ8gOHnmUX9ltIzJ62s4L5zM5C1si6qGAjDcqcOroJVxxrEu1u3VKEAKLaONyg08Bh5Xfv2jDv5qJzvpCU7ym95JUzwxHMOrBHWF8kDz7AToip0yNRxI43qC5xBVgtT9AV6Tu1Tz3qwr4AHf1Ym9h/8Mb4e/DF/pQzZ515voi/Rr/v6rNQ0oFB/4+sX1B+F/bFpf9L+oZesQn8n1H95UG+ujP1HVVm7qR3NB+fF3GOjbJwfw85vQoJ+vrDcNtzS/HHPTvQlXNCedWHhlPaHFpf2S5ZAQjtB+vh/POZpIn/f77j/Z4a40MMdAvsAyfvJfIBR3TNTr5Cr5IGK9umqHRBy59wuYXzAe51yfnsD/+dPni9Wt7Uh:BD69`
can not be analyzed by BinaryKits.
Do you have any idea to solve it?
Thank you!

**Comment by primo-ppcg** [comment]
The `^GF` command and [other applicable commands](https://docs.zebra.com/content/tcm/us/en/printers/software/zebra-zpl-ii,-zbi-2,-set-get-do,-mirror,-wml-programming-guide/c-zpl-zb64-encoding-zb64-encoding-compression/c-zpl-zb64-encoding-b64-z64-encoding.html) do not currently support Z64 encoding, although support could possibly be added as a future enhancement.

To solve your immediate problem, if you could export without compression, that would probably work.

**Reply by masternan** [reply]
Thanks for your idea,may be it is a quick solution。

**Reply by primo-ppcg** [reply]
Support for B64 and Z64 was added in #169.

**Reply by masternan** [reply]
My problem has sloved,thanks @primo-ppcg
